README.md

```diff
@@ -15,83 +16,212 @@ npm run test
 
 This project works a bit differently than a regular test suite for now. We have Jest installed more as a test runner, but we don't really use assertions for now. We use it more to gather and report numbers.
 
+### FCP
+
+First Contentful Paint (FCP) "measures the total time taken from the beginning of a page load to the point any content is rendered on the screen" (see https://web.dev/fcp/).
+To get FCP we use the `PageLoadingMetrics` function from `TraceModel`.
+
+### DCL
+
+DOM Content Loaded (DCL) "marks the point when both the DOM is ready and there are no stylesheets that are blocking JavaScript execution" (see https://web.dev/critical-rendering-path-measure-crp/). In other words, it measures the time when the HTML document has been completely loaded and read.
+To get DCL we use the `PageLoadingMetrics` function from `TraceModel`.
+
+### LCP
+
+The Largest Contentful Paint (LCP) metric "reports the render time of the largest image or text block visible within the viewport, relative to when the page first started loading" (see https://web.dev/lcp/).
+To get LCP we use the `PageLoadingMetrics` function from `TraceModel`.
+
 ### FPS
 
+Frames per second (FPS) measures video quality (how many images are displayed consecutively each second). We use it to measure component animation performance.
 To gather average FPS time, we just use the `FPS` function from `TraceModel`.
-For FPS we don't need to specify any components to look for as it will just grab the SPF for the entire page.
+For FPS we don't need to specify any components to look for as it will just grab the FPS for the entire page.
 
-### Update
-To get React Update Times we just use the `Update` function from `TraceModel`. For update we need to specify the top component in the tree that belongs to the component you're trying to test.
+### FID
 
-Example: Picker is wrapped by numerous Higher-order Components, and the top on is `Changeable`. So we'll use `Update(filename, 'Changeable');` to get the average update times.
+First Input Delay (FID) measures "the time from when a user first interacts with a page (i.e. when they click a link, tap on a button, or use a custom, JavaScript-powered control) to the time when the browser is actually able to begin processing event handlers in response to that interaction" (see https://web.dev/fid/).
+To get FID we use the `FID` function from `TraceModel`.

```

Comment:
FID si calculated using the [PerformanceObserver](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceObserver) interface. Its observer() method specifies the set of entry types to observe.(in this case `first-input`) The performance observer's callback function will be invoked when a performance entry is recorded for one of the specified entryTypes